ui/base_list_ui.py

```diff
@@ -46,32 +45,44 @@ class BaseListUIElement(object):
 
     config_key = "base_list_ui"
 
-    def __init__(self, contents, i, o, name=None, entry_height=1, append_exit=True, exitable=True, scrolling=True, config=None, keymap=None):
+    def __init__(self, contents, i, o, name=None, entry_height=1, append_exit=True, exitable=True, scrolling=True,
+                 config=None, keymap=None):
         self.i = i
         self.o = o
         self.entry_height = entry_height
         self.keymap = keymap if keymap else {}
         self.name = name
         self.append_exit = append_exit
         self.exitable = exitable
-        self.scrolling={"enabled":scrolling,
-                        "current_finished":False,
-                        "current_scrollable":False,
-                        "counter":0,
-                        "pointer":0}
+        self.scrolling = {
+            "enabled": scrolling,
+            "current_finished": False,
+            "current_scrollable": False,
+            "counter": 0,
+            "pointer": 0
+        }
+        self.views = {
+            "TextView": TextView,
+            "EightPtView": EightPtView,
+            "SixteenPtView": SixteenPtView,
+            "MainMenuTripletView": MainMenuTripletView,
+            "PrettyGraphicalView": SixteenPtView,  # Not a descriptive name - left for compatibility
+            "SimpleGraphicalView": EightPtView  # Not a descriptive name - left for compatibility
+        }
         self.config = config if config is not None else global_config
         self.set_view(self.config.get(self.config_key, {}))
         self.set_contents(contents)
         self.generate_keymap()
+        self.view = None
```

Comment:
You still got that leftover from previous commits, thus, it seems, it still hasn't been tested =) I'll look into having CI on GitHub, so that tests could be ran automatically.